site.yml

```diff
@@ -15,6 +15,9 @@ content:
       branches: antora
       start_path: docs
 
+runtime:
+  cache_dir: ./.cache/antora
```

Comment:
This migh prevent us from using build wide cache on CI.

Reply:
Yes, true. We do not need it I just thought it might be good to have everything within project scope

Reply:
How about parallel builds on CI do they share the same cache? Might this get us into trouble?

Reply:
I think not with yarn and PnP, it uses hashes in module cache and the cache is idempotent.

Reply:
The Antora cache is more about the fetched github repos for Camel and Camel K but these will also not get us into trouble when used from multiple builds I guess

Reply:
Oh right, forgot we were talking about Antora cache, we place that cache within the build workspace, which is used by only one build at the time.

https://github.com/apache/camel-website/blob/3b8f70a694952a960505398e16affcd43a17fe83/Jenkinsfile#L39

Similar for GitHub Actions WIP

Reply:
I see so no need for the local cache